### ISSUE TITLE
Inherit image from alpine for making the siz small

### ISSUE THREAD

**Issue by dbjpanda** [dbjpanda/drupal-on-docker#13]
https://github.com/dbjpanda/drupal-on-docker/blob/fea3ead5414653985ef6bed3051d25e3777820ae/docker/php/Dockerfile#L1

And don't add unnecessary packages rather than required. E.g I don't think we will need git on server or on development side.

**Comment by ankitjain28may** [comment]
We require git on php image as we are installing composer in this container and download dependencies using composer and composer use git clone to download some dependencies, i guess it is drush or drupal i am not sure, but composer needs git so i have installed git in that container.

**Comment by Chiranjeeb2410** [comment]
@ankitjain28may what else needs to be done for this as of now?

**Comment by ankitjain28may** [comment]
@Chiranjeeb2410 Need to use alpine image and make all the necessary changes as per alpine image in installing the dependencies.

**Comment by dbjpanda** [comment]
@ankitjain28may Yes, Git is required as composer is download drupal code from git. Well Aim is to minimize the size. Not to install/upgrade/update unnecessarily packages. If we want something else for project specific, then we have already dockerfile, just need to add a line.
